.github/workflows/pipeline.yml

```diff
@@ -148,3 +148,30 @@ jobs:
           project_id: 'logflare-staging'
       - name: 'Trigger Cloud Build'
         run: 'gcloud beta builds triggers run logflare-app-staging-trigger --branch=staging --format "value(name)"'
+  trigger_cloud_build_production:
+    name: Trigger Cloud Build in Production
+    if: github.ref == 'refs/heads/staging'
+    needs:
+      - publish_latest
+      - publish_version
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
```

Comment:
just checking, we need py for gcp sdk right?

Reply:
yes 👍 it's an annoyance 😓